src/machine.tex

```diff
@@ -1723,8 +1723,9 @@ \subsection{Machine Exception Program Counter ({\tt mepc})}
 other invalid addresses prior to writing them to {\tt mepc}.
 
 When a trap is taken into M-mode, {\tt mepc} is written with the virtual
-address of the instruction that encountered the exception.  Otherwise,
-{\tt mepc} is never written by the implementation, though it may be
+address of the instruction that encountered the exception or the virtual
+address of the next instruction to be executed in the interrupted context.
```

Comment:
@kasanovic should verify, but I think with the new proposed terminology, it would suffice to use the existing text, but replace "exception" with "trap."